Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1350
+@@||log.ngsm.nexon.com^|
```

Comment:
I guess that it should be moved to `exclusions.txt`, similar like it was in this case - https://github.com/AdguardTeam/AdGuardSDNSFilter/pull/1304

Reply:
Fixed.

Reply:
I did not face an incorrect blocking issue when a filter blocks it (not DNS filter).
Do you want me to recheck it?

Reply:
Ok. No.